source/VisTrace.cpp

```diff
@@ -451,6 +451,16 @@ LUA_FUNCTION(TraceResult_Pos)
 	LUA->PushVector(MakeVector(pResult->GetPos().x, pResult->GetPos().y, pResult->GetPos().z));
 	return 1;
 }
+
+LUA_FUNCTION(TraceResult_Incident)
+{
+	LUA->CheckType(1, TraceResult::id);
+	TraceResult* pResult = LUA->GetUserType<TraceResult>(1, TraceResult::id);
```

Comment:
This shouldn’t negate, the idea is to pipe it straight into either the VisTrace BSDF or a custom one

Reply:
Caught the wrong line